### PR TITLE
Setup challenge 2021 04 06

### PR DESCRIPTION
**Pull request by ericwburden**
Please take a look and (a) make sure you can run the tests and (b) please review for stupid Java mistakes.

### CODE REVIEW

**Comment by ericwburden** [comment]
Ignore the README changes, I forgot to pull before I pushed.

**Comment by ericwburden** [comment]
Oh, you know what, I symlinked 'mob.java' to the actual source file. Not sure if that works on Windows.

**Comment by ericwburden** [comment]
Ok, I'm done pushing, I promise. And, I looked it up, symlinks are supposed to work on Windows 10. I'd still appreciate anyone pulling and running so it doesn't just "work on my machine".

**Review by parhaml — COMMENTED** [review]
These instructions saying 

`Add your code to the mob.java file`

Can you elaborate? Will each submission add their code to the same file?

**Comment by ericwburden** [comment]
Good question. I was thinking about the mobbing version. TBH, Java makes it a bit more complicated (for me, at least). I'd normally just put the code/function and tests in the same file, but with Java depending on Junit to run the tests, and using Maven as the test runner, I end up with these nested folders for the Code/Test files. I don't know if those folders are required, but I left them as-is and just symlinked the `mob.java` to the `InterveningVerbiage.java` file, and `tests.java` to the `InterveningVerbiageTest.java` file. I'm 100% open to a better structure for this.

**Comment by ericwburden** [comment]
I should also add instructions for the personal solutions, though. Honestly, that's probably more important.

**Comment by parhaml** [comment]
> I should also add instructions for the personal solutions, though. Honestly, that's probably more important.

I agree. For most people these will be most important.

File: challenges/2021-04-06-intervening-verbiage/challenge.md [hunk 1]
```diff
@@ -23,7 +23,13 @@ count_words_between("an", "outhouse", list_of_words)    // 6
 
 ## Tackling This Challenge
 
-To complete this challenge, add your code to the 'mob.java' file. Confirm your solution by running tests by executing the `run_tests.sh` script (use `./run_tests.sh` from the challenge root directory.
+1. Clone (or pull if you've cloned already) the codeconnector/CodeingDojo repository using `git clone https://github.com/codeconnector/CodingDojo.git` or `git pull`, as appropriate.
```
Comment (parhaml) [inline challenges/2021-04-06-intervening-verbiage/challenge.md:26]: We were having people besides us `fork` the repo, correct?

Otherwise, nice job.

Comment (ericwburden) [reply challenges/2021-04-06-intervening-verbiage/challenge.md:26]: Makes sense, I'll update

**Review by parhaml — APPROVED** [review]

**Review by birenderjit — APPROVED** [review]

**Pull request merged**